app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -16,27 +16,47 @@ calls_to_action:
       name: simple
       arguments:
         icon: "icon-arrow"
-        text: "Use the student finance calculator on GOV.UK to find out how much funding you can get."
+        text: "Use the student finance calculator on GOV.UK to get an estimate of how much funding you may get."
         link_text: "Student finance calculator"
         link_target: "https://www.gov.uk/student-finance-calculator"
 navigation: 20.10
 navigation_description: Find out which loans you could get to help pay your fees and living expenses while you train.
 keywords:
-    - Tuition Fee
+    - Tuition Fee Loan
     - Maintenance Loan
     - Student Loan
     - Student Finance
     - Financial Support
 ---
 
-Funding is available to help you train to teach. You can apply for a:
+Funding is available to help you train to teach. You may be eligible for a student loan to cover the cost of your tuition fees or to help with your living costs. 
 
-- tuition fee loan of up to £9,250 to cover your teacher training, so you do not need to pay course fees upfront
-- maintenance loan of up to £13,022  to help with living costs
+You can still apply for a tuition fee and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship. And you will only have to make loan repayments once you’re earning.
 
-You can still apply for a tuition fee and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship.
+You do not need to wait until you have a confirmed offer for a teacher training course to apply for student funding. 
+
+## Tuition fee loans
+You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course so you do not have to pay your course fees upfront. If you’re eligible for a tuition fee loan the amount you borrow is paid directly to your course provider.
+
+The maximum tuition fee loan for a UK applicant:
+
+* full-time postgraduate teacher training course is £9,250 per year 
```

Comment:
```suggestion
* £9,250 per year for a full-time postgraduate teacher training course
```
would it make more sense to front-load with the number, which also means line 41 would make more sense?